src/mlpack/methods/random_forest/random_forest.hpp

```diff
@@ -168,6 +168,8 @@ class RandomForest
    * @param minimumGainSplit Minimum gain for splitting a decision tree node.
    * @param maximumDepth Maximum depth for the tree.
    * @param dimensionSelector Instantiated dimension selection policy.
+   * @param warmStart When set to `true`, it fits new trees and add them to the
+   *     previous forest else a new forest is trained from scratch.
```

Comment:
It might be more clear to write `it adds numTrees new trees to the existing random forest` instead of `it fits new trees and add them to the previous forest`; specifically, it clarifies how many new trees are trained and points out the variable that controls that.

Reply:
Yes, this makes more sense. Thanks 👍